blockchain/blockchain-wallet-v4-frontend
# fix - change password validators moved to wrapper, because of redux-form limitation

## Description
Redux form bug does not allow to create scopes upon rendering https://github.com/erikras/redux-form/issues/3405
Did not investigate why, but perhaps they just compare prop equality
For a fix I've moved validators from template to the form wrapping component.

## Change Type
Please enter one or more of the following: 
- Bug Fix

## Testing Steps
1. Enter incorrect password in the "current password" field. **After the first blur** it should have red border when wrong password is entered.
2. Enter mismatching password to the password confirmation. It should turn red after the first blur as well.

## Code Checklist
- [+] Code compiles successfully (verified via `yarn start`)
- [+] No lint issues exist (verified via `yarn lint`)
- [+] New and existing unit tests pass (verified via `yarn test`)
- [+] `README.md` and other documentation is updated as needed
